mobile/src/modules/transactions/components/send/SendBurstForm.tsx

```diff
@@ -340,10 +345,22 @@ export class SendBurstForm extends React.Component<Props, SendBurstFormState> {
             </BText>
            </View>
         </View>
-        <View>
-          <BButton loading={loading} disabled={!this.isSubmitEnabled()} onPress={this.handleSubmit}>
-            {i18n.t(transactions.screens.send.title)}
-          </BButton>
+        <View style={styles.swipeButtonContainer}>
+          {this.state.showSubmitButton ? <SwipeButton
```

Comment:
a shorter pattern is: `this.state.showSubmitButton && <SwipeButton ... />`

Reply:
This doesn't always work, because it tries to write {false} into the virtualDOM, which throws an error. It seems hit or miss, not sure the exact cause.